.github/workflows/reusable-build-test-release.yml

```diff
@@ -55,94 +55,98 @@ on:
 permissions:
   contents: read
   packages: read
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: ${{ github.event_name == 'pull_request' }}
 jobs:
   setup-workflow:
     runs-on: ubuntu-latest
     outputs:
-      skip-workflow: ${{ steps.skip-workflow.outputs.skip-workflow }}
-      delay-destroy-ko: ${{ steps.delay-destroy-setup.outputs.delay-destroy-ko }}
-      delay-destroy-ui: ${{ steps.delay-destroy-setup.outputs.delay-destroy-ui }}
-      delay-destroy-modinput_functional: ${{ steps.delay-destroy-setup.outputs.delay-destroy-modinput_functional }}
-      delay-destroy-escu: ${{ steps.delay-destroy-setup.outputs.delay-destroy-escu }}
-      delay-destroy-scripted_inputs: ${{ steps.delay-destroy-setup.outputs.delay-destroy-scripted_inputs }}
-      delay-destroy-requirement_test: ${{ steps.delay-destroy-setup.outputs.delay-destroy-requirement_test }}
```

Comment:
Why did you decide to remove separate `delay-destroy-*` and have only 1 variable to control everything?

Reply:
I think you are changing the behaviour here, the original implemented says if you put "preserve: knowledge, ui" in the PR description, it will preserve infra for those 2 tests, after your change, there is no way to do so.

Check https://github.com/splunk/addonfactory-workflow-addon-release/pull/148/files#diff-bdb402178d2b76aa0f8acc15af0ee19e0bd76fd53028475dabe5503b5fe874dcL86-L93 this one, it reads the body of the PR and then preserves some particular infrastructure.

We can rethink the original implementation approach, but let's do this in a separate PR as well, let's just release a label feature, so we don't need to debug other types of issues/

Reply:
I am not sure what should we do with this feature right now, I never used it myself to be honest. We will need to ask people who are using this feature about their feedback.

I like that you made overall code simpler here, I think this is the way we need to go with the workflow, it became too big, but let's focus on smaller iterations.

Let's merge label feature, test it, release it and then come back to all other improvements you proposed in this PR.